include/baresip.h

```diff
@@ -179,6 +179,7 @@ enum call_event {
 	CALL_EVENT_TRANSFER,
 	CALL_EVENT_TRANSFER_FAILED,
 	CALL_EVENT_MENC,
+	CALL_EVENT_ARRIVED,
```

Comment:
I think "arrived" is the wrong wording to use here. What has arrived ?

If the event is sent before incoming, perhaps pre-incoming is a better word ?